test/integration_tests/test_models_correctness.py

```diff
@@ -0,0 +1,124 @@
+# Authors: Bruno Aristimunha
+#
+# License: BSD-3
+
+import mne
+import numpy as np
+import pytest
+import torch
+from mne.io import concatenate_raws
+from skorch.callbacks import LRScheduler
+from skorch.helper import predefined_split
+
+from braindecode.classifier import EEGClassifier
+from braindecode.datasets.xy import create_from_X_y
+from braindecode.models.biot import BIOT
+from braindecode.util import set_random_seeds
+
+
+@pytest.fixture()
+def real_data():
+
+    subject_id = 1
+    # 5,6,7,10,13,14 are codes for executed and imagined hands/feet
+    event_codes = [5, 6, 9, 10, 13, 14]
+
+    # This will download the files if you don't have them yet,
+    # and then return the paths to the files.
+    physionet_paths = mne.datasets.eegbci.load_data(
+    subject_id, event_codes, update_path=False
+    )
+
+    # Load each of the files
+    parts = [
+    mne.io.read_raw_edf(
+        path, preload=True, stim_channel="auto", verbose="WARNING"
+    )
+    for path in physionet_paths
+    ]
+
+    # Concatenate them
+    raw = concatenate_raws(parts)
+
+    # Find the events in this dataset
+    events, _ = mne.events_from_annotations(raw)
+
+    # Use only EEG channels
+    eeg_channel_inds = mne.pick_types(
+    raw.info, meg=False, eeg=True, stim=False, eog=False, exclude="bads"
+    )
+
+    # Extract trials, only using EEG channels
+    epoched = mne.Epochs(
+    raw,
+    events,
+    dict(hands=2, feet=3),
+    tmin=1,
+    tmax=4.1,
+    proj=False,
+    picks=eeg_channel_inds,
+    baseline=None,
+    preload=True,
+    )
+
+    # Convert data from volt to millivolt
+    # Pytorch expects float32 for input and int64 for labels.
+    X = (epoched.get_data() * 1e6).astype(np.float32)
+    y = (epoched.events[:, 2] - 2).astype(np.int64)  # 2,3 -> 0,1
+    return X, y
+
+def test_correctness_biot(real_data):
```

Comment:
I think you will like this test @tgy

Reply:
how long does it take to run? it's not too slow for CIs?

Reply:
12 seconds.

The dataset generation is taking longer from python 3.8:

19.85s setup    test/integration_tests/test_models_correctness.py::test_correctness_biot
12.11s call     test/integration_tests/test_models_correctness.py::test_correctness_biot

Reply:
At the moment, I think we are fine with time in the test and doc, I spent some time optimizing the CI during the last sprint.